index.ts

```diff
@@ -87,8 +94,30 @@ function* identifiers(value) {
     }
 }
 
+function convertMarkdown(markdown: string) {
+    const mdTree = unified().use(remarkParse).parse(markdown);
+    const htmlTree = unified().use(remarkRehype).runSync(mdTree);
+    const text = hastTreeToString(htmlTree);
+
+    let html = unified().use(rehypeStringify).stringify(htmlTree);
+    // Remove leading <p> and trailing </p> if there is only one of each in the
+    // description. (If there are multiple paragraphs, let them be.)
+    if (html.lastIndexOf('<p>') === 0 && html.indexOf('</p>') === html.length - 4) {
+      html = html.substring(3, html.length - 4);
+    }
+
+    return { text, html };
+}
+
 const features: { [key: string]: FeatureData } = {};
 for (const [key, data] of yamlEntries('feature-group-definitions')) {
+    // Convert markdown to text+HTML.
+    if (data.description) {
+        const { text, html } = convertMarkdown(data.description);
+        data.description = text;
+        data.descriptionHTML = html;
```

Comment:
For all our other fields we use `_` as a word separator. Maybe this instead?

```suggestion
        data.description_html = html;
```